blog/2021-08-09-zebra-medical-case-study.md

```diff
@@ -0,0 +1,101 @@
+---
+tags:
+  - v1
+  - announcement
+  - Zebra Medical Vision
+  - Data pipeline
+  - ETL
+  - Machine learning
+  - microservice-orchestration
+  - microservices
+  - case study
```

Comment:
personally think we gotta chill a bit with the tags. too many tags looks desperate.

```suggestion
  - Zebra Medical Vision
  - Data pipeline
  - Machine learning
  - microservice-orchestration
  - case study
```